kernel/kernel-api/src/test/scala/io/delta/kernel/internal/checkpoints/CheckpointerSuite.scala

```diff
@@ -239,6 +236,20 @@ object CheckpointerSuite {
     override def getSize: Int = 1
   }
 
+  val SAMPLE_SIDECAR_FILE_CONTENT: ColumnarBatch = new ColumnarBatch {
```

Comment:
is there a test in this suite that uses this?

Reply:
No, removed.